lm75bd/lm75bd.c

```diff
@@ -27,13 +30,25 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+
+  // set pointer register to temperature
+  uint8_t point_reg = 0x00;
+  uint8_t buf[1] = {point_reg};
+  i2cSendTo(devAddr, buf, 1);
```

Comment:
use `RETURN_IF_ERROR_CODE` to make sure this error gets dealt with if `i2cSendTo` fails